versions/1/kibana/spec.yml

```diff
@@ -65,3 +65,25 @@
         type: file
         contentMediaType: "application/json"
         pattern: '^.+\.json$'
+    - description: Folder containing rules
+      type: folder
+      name: "rules"
```

Comment:
Is `rules` the saved object type? Thinking out loud: Is this `security-rules`? Lets stick to the SO type.

Reply:
Went with `security-rule`. These folder names are typically in singular form right?

Reply:
yes. And I think in most cases we use `_` and not `-`. @mtojek ?

Reply:
That looks right, updated to `security_rule`